ui/src/App.tsx

```diff
@@ -64,34 +64,50 @@ function App() {
           </Stack>
         </Toolbar>
       </AppBar>
-      <Container sx={{ py: 2, flex: 1 }}>
-        <Typography variant="h4">Hello</Typography>
-        <Typography variant="body1">Explanation stuff</Typography>
-        <Box
-          sx={theme => ({
-            display: 'grid',
-            gridTemplateColumns: '1fr 1fr 1fr',
-            gridAutoRows: '1fr',
-            width: '600px',
-            height: '600px',
-            border: '3px solid black',
-            borderRadius: theme.shape.borderRadius,
-            overflow: 'hidden',
-            '& > button': {
-              backgroundColor: 'grey',
-              borderRadius: "0px",
-              '&:not(:nth-child(3n))': {
-                borderRight: 'thin solid black',
-              },
-              '&:nth-child(-n+6)': {
-                borderBottom: 'thin solid black',
-              },
-            },
-          })}
-        >
-          {[...Array(9).keys()].map(entry => <GridButton key={entry}/>)}
-        </Box>
-      </Container>
+        <Container sx={{ py: 2, flex: 1}}>
+          <Typography variant="h4" sx = {{textAlign: 'center'}}>Hello</Typography>
+          <Typography variant="body1" sx = {{textAlign: 'center'}}>Explanation stuff</Typography>
+          <Stack alignItems='center'>
+            <Stack direction='row' alignItems={'center'}>
+              <Typography variant="body1" sx={{textAlign:'center', width:'200px'}}>A category</Typography>
+              <Typography variant="body1" sx={{textAlign:'center', width:'200px'}}>Some other category</Typography>
+              <Typography variant="body1" sx={{textAlign:'center', width:'200px'}}>Looooooooong looooooong multi-line category</Typography>
```

Comment:
I'd use maxWidth instead. Also consider percentages or `vw` metrics. Alternatively, simply set the padding/margin to whatever feels comfortable and let the browser figure out the rest. Not awful tbh

Reply:
Why is maxWidth here relevant? My main concern is in ensuring that each component takes up exactly 200px. If I were to use maxWidth, I would have no guarantee that components actually do take up the full 200px that I need them to occupy.

Reply:
Mobile support - 200px on mobile is treated very differently than 200px on desktop.